docker-compose.yaml

```diff
@@ -47,7 +47,7 @@ services:
       # Alias the hostname to avoid underscores in domain names, as that causes the following error in cla_backend:
       # "Invalid HTTP_HOST header: 'cla_backend'. The domain name provided is not valid according to RFC 1034/1035"
       - "cla_backend:cla-backend"
-  cla_frontend:
+  case_handling_system:
```

Comment:
It seems to be useful to keep the application name to `cla_frontend` as it matches the github repository. What do you think?